lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -46,6 +50,11 @@ const char kAddAnonSublayerFlagL[] = "addAnonymous";
 const char kMuteLayerFlag[] = "mt";
 const char kMuteLayerFlagL[] = "muteLayer";
 
+const char kAnyDirtyLayerFlag[] = "ad";
+const char kAnyDirtyLayerFlagL[] = "anyDirty";
+
+std::set<unsigned int> _supportedTypesForDirtyQuery;
```

Comment:
Since the method that add/remove from this set are static shouldn't this also be a static data member?

Reply:
I don't think it's required because this is declared in the unnamed namespace?